LoRaEngine/modules/LoRaWanNetworkSrvModule/LoraTools/Regions/LoRaRegionType.cs

```diff
@@ -11,6 +11,7 @@ public enum LoRaRegionType
         CN470RP1,
         CN470RP2,
         AS923,
+        AU915,
```

Comment:
Do we need to add the AU concentrator file to the lora-cli folder?

Reply:
Great point, adding it